src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="form_field">
+      <label class="search_bar search1" >
+        <img src="./images/Search.svg" alt="search image" 
+        class="image_big">
```

Comment:
Check the code style, please. Don’t add words like image or icon to the alt atribute.
```suggestion
        <img 
          src="./images/Search.svg" 
          alt="search" 
          class="search-icon search-icon--big"
        >
```

Reply:
done